src-ts/types.ts

```diff
@@ -0,0 +1,28 @@
+export interface DeployedContracts {
+  l1UpgradeExecutorLogic: string;
+  l2TimelockLogic: string;
+  l2GovernorLogic: string;
+  l2FixedDelegateLogic: string;
+  l2TokenLogic: string;
+  l2UpgradeExecutorLogic: string;
+  l1GovernanceFactory: string;
+  l2GovernanceFactory: string;
+  l1ReverseCustomGatewayLogic: string;
+  l1ReverseCustomGatewayProxy: string;
+  l2ReverseCustomGatewayLogic: string;
+  l2ReverseCustomGatewayProxy: string;
```

Comment:
any reason to not just store the proxy address and fetch the logic in runtime when needed? (doesn't seem like we currently use it)

Reply:
actually if i'm understanding corrrectly, the gov deployments stores them for in-progress deploys so it can resume later; gonna leave as is for now